com.unity.perception/Runtime/GroundTruth/Labelers/BoundingBoxLabeler.cs

```diff
@@ -108,8 +116,73 @@ void OnRenderedObjectInfosCalculated(int frameCount, NativeArray<RenderedObjectI
                     };
                 }
 
+                if (!CaptureOptions.useAsyncReadbackIfSupported && frameCount != Time.frameCount) 
+                    Debug.LogWarning("Not on current frame: " + frameCount + "(" + Time.frameCount + ")");
+
+                if (IsVisualizationEnabled()) 
+                {
+                    Visualize();
+                    ReportFrameCount(frameCount);
```

Comment:
Probably the 2d bounding box labeler is not the right place to report the frame count.

Reply:
Yeah, that was something that I put in for testing and forgot to remove. It's gone now.